lib/src/components/position_component.dart

```diff
@@ -129,7 +128,7 @@ abstract class PositionComponent extends BaseComponent {
   @override
   void renderDebugMode(Canvas canvas) {
     if (this is Hitbox) {
-      (this as Hitbox).renderContour(canvas);
+      (this as Hitbox).renderShapes(canvas);
     }
     canvas.drawRect(size.toRect(), debugPaint);
```

Comment:
should this be done only instead (else) of the first render?

Reply:
This one is pretty handy to have too, because then you can make sure your shapes are within the boundaries.